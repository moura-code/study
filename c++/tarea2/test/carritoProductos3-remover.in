crearCarritoProductos
crearFecha 1/2/2001
crearProducto 125 Silla 9
agregarCarritoProductos
removerDeCarritoProductos 125
imprimirCarritoProductos
crearFecha 1/4/2001
crearProducto 12 Mesa 9
agregarCarritoProductos
crearFecha 1/1/2001
crearProducto 24 Sillon 9
agregarCarritoProductos
crearFecha 1/5/2001
crearProducto 15 Computadora 9
agregarCarritoProductos
imprimirCarritoProductos
removerDeCarritoProductos 15
imprimirCarritoProductos
removerDeCarritoProductos 24
imprimirCarritoProductos
removerDeCarritoProductos 12
imprimirCarritoProductos
liberarCarritoProductos
Fin
