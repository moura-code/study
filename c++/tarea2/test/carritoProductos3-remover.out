Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Carrito de productos creado de forma exitosa.
2> Fecha creada en forma exitosa.
3> Producto creado en forma exitosa.
4> Producto agregado de forma exitosa.
5> Producto con id 125 removido con exito.
6> 7> Fecha creada en forma exitosa.
8> Producto creado en forma exitosa.
9> Producto agregado de forma exitosa.
10> Fecha creada en forma exitosa.
11> Producto creado en forma exitosa.
12> Producto agregado de forma exitosa.
13> Fecha creada en forma exitosa.
14> Producto creado en forma exitosa.
15> Producto agregado de forma exitosa.
16> Producto: 12
Mesa
Precio: 9
Ingresado el: 1/4/2001
Producto: 15
Computadora
Precio: 9
Ingresado el: 1/5/2001
Producto: 24
Sillon
Precio: 9
Ingresado el: 1/1/2001
17> Producto con id 15 removido con exito.
18> Producto: 12
Mesa
Precio: 9
Ingresado el: 1/4/2001
Producto: 24
Sillon
Precio: 9
Ingresado el: 1/1/2001
19> Producto con id 24 removido con exito.
20> Producto: 12
Mesa
Precio: 9
Ingresado el: 1/4/2001
21> Producto con id 12 removido con exito.
22> 23> Carrito de productos liberado con exito.
24> Fin.
